admin/metabox/class-metabox.php

```diff
@@ -596,6 +596,9 @@ public function do_meta_box( $meta_field_def, $key = '' ) {
 				$content .= '<label for="' . $esc_form_key . '" class="screen-reader-text">' . esc_html( $meta_field_def['label'] ) . '</label>';
 				$content .= '<input type="text"' . $placeholder . ' id="' . $esc_form_key . '" autocomplete="off" name="' . $esc_form_key . '" value="' . esc_attr( $meta_value ) . '" class="large-text' . $class . '"/>';
 
+				$button = new WPSEO_Metabox_Keyword_Synonyms_Button();
```

Comment:
Please wrap this with `WPSEO_Utils::is_yoast_seo_premium()` to make sure this call to action doesn't appear in Premium.